src/message.js

```diff
@@ -108,7 +112,9 @@ Globalize.prototype.messageFormatter = function( path ) {
  */
 Globalize.formatMessage =
 Globalize.prototype.formatMessage = function( path /* , variables */ ) {
-	return this.messageFormatter( path ).apply( {}, slice.call( arguments, 1 ) );
+	return ( arguments[ 1 ] && typeof arguments[ 1 ] === "boolean" ) ?
+		this.messageFormatter( path, arguments[ 1 ] ).apply( {}, slice.call( arguments, 2 ) ) :
+		this.messageFormatter( path ).apply( {}, slice.call( arguments, 1 ) );
```

Comment:
The example is just as I pointed at the beginning of this thread.
You may add in your 
\globalize\examples\plain-javascript\index.html the following:
	Globalize.loadMessages({
		en: {
			breadcrumb: "{0} >> {1} >> {2} >> {3}",
			}
		}		
	});
	var fmOut = Globalize( "en" ).formatMessage( "breadcrumb", "first", "second", "third", "forth"  );